run-tests.sh

```diff
@@ -11,6 +11,18 @@ export PATH=$PREFIX/bin:$PATH
 
 : ${ARCHS:=${TOOLCHAIN_ARCHS-i686 x86_64 armv7 aarch64}}
 
+if [ -z $RUN_INTEL ]; then
```

Comment:
LGTM in general, but I'd prefer to have it named `RUN_X86`. And in this particular line I'd like quotes around it, like `if [ -z "$RUN_X86" ]; then`.

Reply:
Done